main.cpp

```diff
@@ -42,7 +42,134 @@ send me a DM to check your pull request
  */
 
 #include <iostream>
+
+
+
+struct FloatType
+{
+    float add( float lhs, float rhs );
+    float subtract( float lhs, float rhs );
+    float multiply( float lhs, float rhs );
+    float divide( float lhs, float rhs );
+};
+
+float FloatType::add ( float lhs, float rhs)
+{
+    return ( lhs + rhs);
+}
+
+float FloatType::subtract ( float lhs, float rhs)
+{
+    return ( lhs - rhs);
+}
+
+float FloatType::multiply ( float lhs, float rhs)
+{
+    return ( lhs * rhs);
+}
+
+float FloatType::divide ( float lhs, float rhs)
+{
+    if (rhs > 0.f) 
+    {
+        return ( lhs / rhs); 
+    }
+    std::cout << "Watch out! Float Divison by 0." << std::endl;
+    return ( lhs / rhs);
```

Comment:
You have some logic errors here. 
what happens if `rhs` is negative? 
Read up on `std::abs()` and epsilon.   
- [ ] revise to use correct logic.